python/CMakeLists.txt

```diff
@@ -7,18 +7,26 @@ file(GLOB UTILS_PY_FILES . ./paddle/utils/*.py)
 file(GLOB_RECURSE V2_PY_FILES ./paddle/v2/ *.py)
 
 set(PY_FILES paddle/__init__.py
-             ${TRAINER_PY_FILES}
-             ${HELPERS_PY_FILES}
-             ${UTILS_PY_FILES}
-             ${V2_PY_FILES})
+  ${TRAINER_PY_FILES}
+  ${HELPERS_PY_FILES}
+  ${UTILS_PY_FILES}
+  ${V2_PY_FILES})
+
+add_custom_target(copy_paddle_master)
+if(WITH_GOLANG)
+  add_custom_command(TARGET copy_paddle_master
```

Comment:
Thanks! Done.
Just curios, are there any problem with the old way, or it's just `configure_file` is more ideomatic for copying files.